### DISCUSSION TITLE
Love the style, but one question

### DISCUSSION THREAD

**Discussion by landjam** [anthony1x6000/ROBLOX2016stylus#3] [Q&A] [answered]
The plugin looks amazing and I hope you keep on improving it and keeping it up to date, but I have been having this weird problem with the style. For some reason, it changes all of my sites to this style when I have it on. This is quite annoying as it makes google.com look strange. Do you know any fixes for this?

Thanks,
Ham

**Comment by anthony1x6000** [comment, marked as answer]
Thanks for the feedback! To fix your issue go to the edit style page:
![image](https://user-images.githubusercontent.com/33004321/134436348-54301a40-bac9-43eb-807e-e613d5d9ac23.png)
Then at the bottom click the +
![image](https://user-images.githubusercontent.com/33004321/134436377-ae8d3581-eec1-4970-9278-7b06bc8c532c.png)
Then select "URLs on the domain" then `roblox.com` in the text box:
![image](https://user-images.githubusercontent.com/33004321/134436442-12f39709-b5f7-48d7-987a-799ad9fc191f.png)

That should fix your problem. If that doesn't work keep the last fixes and remove line 1 `/* i really want this to be global */`. 
And if that doesn't work at the top of the css put `@-moz-document url-prefix("https://roblox.com"), url-prefix("https://web.roblox.com"){}`

And if you don't want to do all that I just updated the [CSS](https://github.com/anthony1x6000/ROBLOX2016stylus/blob/main/stylustheme.css) on Github and just copy and pasting that should work.

**Reply by landjam** [reply]
I cannot seem to find the plus button on stylus. Would you mind pointing it out to me?
![image](https://user-images.githubusercontent.com/87459429/134741459-a04850b8-3336-4dc2-894e-ca50a10427c6.png)

**Reply by anthony1x6000** [reply]
> I cannot seem to find the plus button on stylus. Would you mind pointing it out to me?
> ![image](https://user-images.githubusercontent.com/87459429/134741459-a04850b8-3336-4dc2-894e-ca50a10427c6.png)

I just updated the style. Hopefully that fixes the problem. [Userstyles ](https://userstyles.org/styles/189624/roblox-2016-games-page-fix-icon-fix )& [Github](https://github.com/anthony1x6000/ROBLOX2016stylus/releases/tag/1.1.9).
I think the fix I answered with only applies if you copied and pasted the style yourself. Sorry for the late reply.

**Reply by landjam** [reply]
It worked. Thanks for all the help.
